tmt/utils.py

```diff
@@ -5047,17 +5140,24 @@ def _load_keys(
                 # try+except seems to work better than get(), especially when
                 # semantic of fmf.Tree.get() is slightly different than that
                 # of dict().get().
```

Comment:
The `try-except` clause has been removed. Please, update the comment as well. Would be also nice to shortly explain the difference between `source_keyname` and `source_keyname_cli`.

Reply:
Fixed in 93b996cb